docs/tutorial/sandbox.md

```diff
@@ -100,17 +93,32 @@ app.whenReady().then(() => {
 })
 ```
 
+Sandboxing is also disabled whenever Node.js integration is enabled in the renderer.
+This can be done through the BrowserWindow constructor with the `contextIsolation: false`
+and `nodeIntegration: true` flags:
```

Comment:
technically `contextIsolation: false` isn't required to enable node integration, and I'd argue that having it on is a good idea as it provides some security barrier!

Reply:
> technically `contextIsolation: false` isn't required to enable node integration

What practical things can you achieve with `nodeIntegration: true` without also adding `contextIsolation: false`? Any `require` call errors out with `Uncaught ReferenceError: require is not defined`.

Reply:
require in the preload, expose specific APIs through `contextBridge`. That way you can (somewhat) limit rogue JS from accessing full Node.js.